frontstage/views/secure_messaging.py

```diff
@@ -160,32 +161,20 @@ def get_survey_id(party_id):
     return survey_name
 
 
-def get_messages(label, logger):
-    headers = {"Authorization": request.cookies['authorization']}
+def get_messages_list(label, logger):
     logger.debug('Attempting to retrieve messages')
-    url = app.config['MESSAGES_API_URL']
-    if label is not None:
-        url = url + "&label=" + label
-    response = requests.get(url, headers=headers)
+    headers = {"Authorization": request.cookies['authorization']}
```

Comment:
Is this something that could be pushed into `api_call` as it seems like a generic header that will (probably) always need to be present?

Reply:
This is only sent in the secure messages requests so isn't a generic to all requests. The other services are using BASIC_AUTH currently, would be good to have this standardised so we can make it generic though